xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -941,23 +896,13 @@ struct UserProfileDetailModel: ModelProtocol {
         return update(
             state: model,
             actions: [
-                .presentRenameSheet(false),
-                .refresh(forceSync: true)
+                .refresh(forceSync: false),
+                .requestWaitForFollowedUserResolution(to) // TODO: refactor
```

Comment:
This refactor will happen in https://github.com/subconsciousnetwork/subconscious/pull/996

Reply:
Assuming this PR is functioning, and the refactor is an improvement?